src/compiler/core.ts

```diff
@@ -1112,7 +1112,7 @@ function stableSortIndices<T>(array: readonly T[], indices: number[], comparer:
  * @internal
  */
 export function sort<T>(array: readonly T[], comparer?: Comparer<T>): SortedReadonlyArray<T> {

```

Comment:
If I rename this to `toSorted`, would that be preferred? This is really just a polyfill for `toSorted` that appears to be slightly faster (for now).

Reply:
sure